templates/views-view/views-view.component.inc

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+/**
+ * Implements hook_preprocess_views_view_grid().
+ */
+function europa_preprocess_views_view_grid(&$variables) {
+  $variables['theme_hook_suggestions'][] =  'grid';
```

Comment:
Yes sorry i misunderstood your question, so what do you suggest to not duplicate the tpl?